pkg/operator/operator_test.go

```diff
@@ -161,3 +161,72 @@ func TestEnsureCertsSelfSigned(t *testing.T) {
 		})
 	}
 }
+
+func TestResourceLimit(t *testing.T) {
+	var (
+		timeout   = 3 * time.Second
+		_, cancel = context.WithTimeout(context.Background(), timeout)
+	)
+	t.Cleanup(cancel)
+
+	dir, err := ioutil.TempDir("", "test_resource_limit")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(dir)
+	type resource struct {
+		collectorCPUResource    int64
+		collectorMemoryResource int64
+		collectorMemoryLimit    int64
+		evaluatorCPUResource    int64
+		evaluatorMemoryResource int64
+		evaluatorMemoryLimit    int64
+	}
+	for _, tc := range []struct {
+		desc             string
+		opts             Options
+		expectedResource resource
+	}{
+		{
+			desc:             "no resource assigned",
+			opts:             Options{ProjectID: "test", Cluster: "test"},
+			expectedResource: resource{collectorCPUResource: 100, collectorMemoryResource: 200, collectorMemoryLimit: 3000, evaluatorCPUResource: 100, evaluatorMemoryResource: 200, evaluatorMemoryLimit: 1000},
```

Comment:
Can you reformat here to have newlines between the struct fields for readability? i.e.
```go
expectedResource: resource{
	collectorCPUResource: 100,
	collectorMemoryResource: 200,
	collectorMemoryLimit: 3000,
	evaluatorCPUResource: 100,
	evaluatorMemoryResource: 200,
	evaluatorMemoryLimit: 1000,
}
```
Same for the other test cases.